source/computer/macos-setup.md

```diff
@@ -72,58 +72,28 @@ Homebrew 的安装脚本及相关资源托管在 [GitHub](https://github.com/) 
 针对国内用户的 Homebrew 安装和配置指南来自于 <https://brew.idayer.com/>。
 :::
 
-安装 Homebrew:
+打开终端，执行如下命令，并根据终端提示进行操作，以安装 Homebrew:
 
 ```
 $ /bin/bash -c "$(curl -fsSL https://gitee.com/ineo6/homebrew-install/raw/master/install.sh)"
 ```
-
-启用 [Homebrew Cask](https://github.com/Homebrew/homebrew-cask) 以通过命令行
-安装带有图形界面的软件（如 VS Code、QQ）并设置使用中科大镜像:
-
-```
-$ brew tap --custom-remote --force-auto-update homebrew/cask https://mirrors.ustc.edu.cn/homebrew-cask.git
-```
-
-设置从中科大镜像下载 bottles （二进制安装包）:
-
-```
-$ echo 'export HOMEBREW_BOTTLE_DOMAIN=https://mirrors.ustc.edu.cn/homebrew-bottles/bottles' >> ~/.zshrc
-$ source ~/.zshrc
-```
```

Comment:
The installation script now uses the USTC mirror by default in
https://github.com/ineo6/homebrew-install/commit/8747a725d68116f56c1558554538f63d53978294